src/middlewared/middlewared/etc_files/ftpusers.mako

```diff
@@ -0,0 +1,11 @@
+<%
+    ftp = render_ctx['ftp.config']
+    users = [user["username"] for user in render_ctx['user.query']]
+
+    exclude = []
+    if ftp["rootlogin"]:
```

Comment:
I think maybe for security reasons we should just remove this option from electric eel and document in release notes (unless you can think of any particular reason it must stay).